pyxem/signals/diffraction2d.py

```diff
@@ -1981,59 +1981,38 @@ def get_azimuthal_integral1d(
         >>> det = Detector(pixel1=1e-4, pixel2=1e-4)
         >>> ds.get_azimuthal_integral1d(npt=100, detector_dist=.2, detector= det, wavelength=2.508e-12)
         """
-        if lazy_result is None:
-            lazy_result = self._lazy
-        signal_type = self._signal_type
-        unit = to_unit(self.unit)
```

Comment:
I'm not too familiar with this `to_unit` function, what did it do. Any why is it removed?

Reply:
I just removed it to be more inline with the azimuthal_integral2D.  I changed it so that we register k as a unit in https://github.com/pyxem/pyxem/blob/e2e0485bd67b8d1c32ae54cfd0e936c2018dfc5c/pyxem/utils/pyfai_utils.py#L151-L171

It doesn't really do anything at this point.